dashboard_description_plugin/webpack.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = (_env, argv) => {
     const protocol = new URL(effectiveBackendUrl).protocol;
 
     const proxy = {
-        "/api": {
+        "/gdc": {
```

Comment:
This should depend on the backend type. In tiger it's `/api`.

Reply:
Thanks for pointing this out, I'll fix it in the internal plugins repo too. Here fixed in https://github.com/gooddata/gooddata-plugin-examples/compare/973348080463185479653b6051b911c299044742..fbc943ef299d1d36c145d642099477acdd4447d6